src/borg/cache.py

```diff
@@ -257,17 +260,20 @@ def commit(self):
                     if entry.age == 0 and bigint_to_int(entry.mtime) < self._newest_mtime or \
                        entry.age > 0 and entry.age < ttl:
                         msgpack.pack((path_hash, entry), fd)
+        pi.output('Saving config cache')
```

Comment:
Saving cache config

(and I'm not really sure this is useful - the following lines are expected to take some milliseconds only)